site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -753,6 +753,52 @@ public class MyAnnotatedService {
 }
 ```
 
+## Specifying a blocking task executor
+
+An annotated service is executed by an [EventLoop](https://netty.io/4.1/api/io/netty/channel/EventLoop.html),
+so you must not make a blocking call within the `EventLoop`. If you want to make a blocking call,
```

Comment:
Do you know any good reference about event loop and blocking call? I could only find about Node.js.
If there's no good reference, I will add about it as well.